### DISCUSSION TITLE
Why is Github Copilot encountering temporary issues?

### DISCUSSION THREAD

**Discussion by officialayangithub** [community/community#8215] [Copilot Conversations]
Tell me, please! I was writing Python code!

**Comment by davecheney** [comment]
@officialayangithub I don't see your github username on the waitlist. Did you receive an email telling you that you were invited to use copilot?

**Reply by shruthibadri** [reply]
hi, I had the same problem - a message stating "temporary issues". and i did see a message on the github site that said my account was approved for copilot

thanks

**Reply by davecheney** [reply]
Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by ghost** [reply]
Hi, I had the same problem. "Copilot is encountering temporary issues" and my account was approved for copilot

**Reply by shruthibadri** [reply]
`[INFO] [ghostText] [2021-11-30T13:00:51.219Z] Cancelled after extractPrompt
[INFO] [default] [2021-11-30T13:00:51.366Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-11-30T13:00:51.447Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-11-30T13:00:52.915Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-11-30T13:02:14.297Z] Breaking, no choices
[INFO] [ghostText] [2021-11-30T13:02:14.297Z] Breaking, no choices
[INFO] [ghostText] [2021-11-30T13:02:14.297Z] Breaking, no choices`

**Reply by davecheney** [reply]
looks like its working

**Reply by shruthibadri** [reply]
when i hover over the copilot icon in the bottom bar, it says "copilot is encountering temporary issues"

**Reply by tanxeel** [reply]
I have the same issue. copilot is encountering temporary issues

**Reply by Astroide** [reply]
Same issue for me, Copilot was working yesterday and then today it does not work anymore and shows this icon:
![Screenshot](https://user-images.githubusercontent.com/65875137/144315739-31d52a71-c360-48c1-9316-2a8da36d7189.png)

**Edit:** I uninstalled Copilot, then reinstalled it, and that fixed the issue.

**Reply by shruthibadri** [reply]
Thanks @Astroide, this worked!

**Reply by m-dehghani** [reply]
Same issue for me!

output in VsCode: 

[INFO] [default] [2021-12-06T07:47:10.915Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-12-06T07:47:11.195Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-12-06T07:47:11.443Z] Breaking, no choices
[INFO] [ghostText] [2021-12-06T07:47:24.433Z] Breaking, no choices

**Reply by davecheney** [reply]
Copilot is working; it just has nothing to offer you at that time.

**Reply by m-dehghani** [reply]
So the error message is misleading

**Comment by rhythmdev** [comment]
Sometimes I face the same issues. To fix this  I just close the Vs code and reopen it or uninstall Copilot and reinstall it. Then it will work for me.

**Comment by shankarlmc** [comment]
just reload the vs code. It works fine.
